_content/articles/temple-where-tattoos-come-to-life_lastrucci-francesco.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Inside the Thai Temple Where Tattoos Come to Life"
+authors:
+  - "Francesco Lastrucci"
+external_url: ""https://www.nytimes.com/2021/10/25/travel/sak-yant-tattoo-thailand.html
+drive_links:
+  - "https://drive.google.com/file/d/1bmMEN9bdURZF1MRDDJQf8PPn2bZ2iE_i/view?usp=sharing"
+course: thai
+tags:
+  - mantra
```

Comment:
🤔 For magic I usually use "iddhi" instead. I think the tantra tag is sufficient here though 😊